package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/prime",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
nit: I don't think we need a bump since we're only adding tests and not PRIME features